### ISSUE TITLE
OpenLiberty 23.0.0.7 with webProfile-8.0 logs messages saying it requires annotations in the jakarta.annotation namespace

### ISSUE THREAD

**Issue by yogregg** [OpenLiberty/open-liberty#25834]
When my Java EE web application war file starts up, I am seeing messages like this in console.log:

```
[AUDIT   ] CWWKM0483I: The javax.annotation.PostConstruct annotation is ignored on classes in the ace-test-liberty-generic module of the ace-test-liberty-generic application. Jakarta EE applications must use the jakarta.annotation.PostConstruct annotation. The affected classes are: <REDACTED>
```

I don't understand why this is happening.  I am using webProfile-8.0 in my server.xml, and the log shows that:

```
[AUDIT   ] CWWKF0012I: The server installed the following features: [appSecurity-2.0, appSecurity-3.0, beanValidation-2.0, cdi-2.0, distributedMap-1.0, ejbLite-3.2, el-3.0, jaspic-1.1, jaxrs-2.1, jaxrsClient-2.1, jdbc-4.2, jndi-1.0, jpa-2.2, jpaContainer-2.2, jsf-2.3, jsonb-1.0, jsonp-1.1, jsp-2.3, managedBeans-1.0, servlet-4.0, ssl-1.0, webProfile-8.0, websocket-1.1].
```

My source code uses javax.annotation.PostConstruct.  I am seeing this for various classes, and also for the javax.annotation.PreDestroy annotation.  I have also tried this with earlier versions (21.0.0.8 and 22.0.0.9) and do NOT see the same messages logged with those.  This is surprising to me because this is a Java EE web application war file that still uses the javax namespace, and it is using webProfile-8.0 which is supposed to support Jakarta EE 8, which specifies the javax namespace, not the jakarta namespace.  What is going on here?

I am using these versions, from the messages.log output:

product = Open Liberty 23.0.0.7 (wlp-1.0.79.cl230720230710-1201)
java.version = 17.0.1
java.runtime = IBM Semeru Runtime Open Edition (17.0.1+12)
os = Windows 10 (10.0; amd64) (en_US)

I downloaded OpenLiberty in its "Web Profile 8" form, and this is my server.xml:

```
<?xml version="1.0" encoding="UTF-8"?>
<server description="new server">

    <!-- Enable features -->
    <featureManager>
        <feature>webProfile-8.0</feature>
    </featureManager>

    <!-- To access this server from a remote client add a host attribute to the following element, e.g. host="*" -->
    <httpEndpoint id="defaultHttpEndpoint"
                  host="*"
                  httpPort="9380"
                  httpsPort="9343" />
                  
    <!-- Automatically expand WAR files and EAR files -->
    <applicationManager autoExpand="true"/>

    <!-- Default SSL configuration enables trust for default certificates from the Java runtime --> 
    <ssl id="defaultSSLConfig" trustDefaultCerts="true" />

    <webContainer deferServletLoad="false"/>
        <mimeTypes>
        <type>svg=image/svg+xml</type>
        <type>svgz=image/svg+xml</type>
        <type>css=text/css</type>
    </mimeTypes>
</server>
```

**Comment by tkburroughs** [comment]
The message occurs when `jakarta.annotation.Resource` is found somewhere on the classpath and the application is using `javax.annotation.xxxx`.

webProvile-8.0 should not be adding that; I recommend checking to see if the jakarta annotation API jar was packaged with the application?  or really anywhere else on the application classpath.

**Comment by yogregg** [comment]
Thank you very much for the quick response.  That jar was in fact packaged with the application, and I've changed it to no longer need that.  With that change, it resolved the problem for me.  So from my perspective, this can be closed.

**Comment by tkburroughs** [comment]
Thank you for confirming this issue was caused by the Jakarta package jar being included in the application.  Glad to hear removing the jar has resolved the issue.

While it is not recommended to include Jakarta API jars in an application (since they are provided by Liberty features), logging this particular message for this scenario is very confusing.  I'm going to go ahead and correct the Liberty code to only log the CWWKM0483I message for the intended purpose, thus avoiding confusion for this scenario.

This will be fixed by PR #25889

**Comment by yogregg** [comment]
Thanks @tkburroughs that's a very helpful change.
